pages/index.mustache

```diff
@@ -0,0 +1,191 @@
+							<button class="nav-link active" id="nav-all-tab" data-bs-toggle="tab" data-bs-target="#nav-all"
+								type="button" role="tab" aria-controls="nav-all" aria-selected="false">All</button>
+              {{#allPossibleCategories}}
+                <button class="nav-link" id="nav-{{name}}-tab" data-bs-toggle="tab" data-bs-target="#nav-{{name}}"
+                  type="button" role="tab" aria-controls="nav-{{name}}" aria-selected="false">{{displayName}}</button>
+              {{/allPossibleCategories}}
+						</nav>
+					</div>
+				</div>
+				<!-- Tab Selector for Small Screens -->
+				<div class="col-12 d-lg-none">
+					<div class="nav-list py-3 px-3">
+						<nav class="nav nav-pills" id="nav-tab" role="tablist" aria-orientation="horizontal">
+							<button class="nav-link active" id="nav-all-tab" data-bs-toggle="tab" data-bs-target="#nav-all"
+								type="button" role="tab" aria-controls="nav-all" aria-selected="false">All</button>
+              {{#allPossibleCategories}}
+                <button class="nav-link" id="nav-{{name}}-tab" data-bs-toggle="tab" data-bs-target="#nav-{{name}}"
+                  type="button" role="tab" aria-controls="nav-{{name}}" aria-selected="false">{{displayName}}</button>
+              {{/allPossibleCategories}}
+						</nav>
+					</div>
+
+				</div>
+
+				<!-- Tab Content -->
+				<div class="col-lg-10 col-12">
+					<div class="tab-content" id="nav-tabContent">
+						<div class="tab-pane fade show active" id='nav-all' role="tabpanel" aria-labelledby='nav-all-tab'
+							tabindex={0}>
+							<!-- Trending Section -->
+							<div class="row mx-3 py-4">
+								<div class="anchor" id="trending-section"></div>
+								<h5 class="py-3">Trending Plugins</h5>
+
+							</div>
+							<!-- Recommended Section -->
+
+							<div class="row mx-3 py-4">
+								<div class="anchor" id="recommended-section"></div>
+								<h5 class="py-3">Recommended Plugins</h5>
+
+							</div>
+							<!-- All Plugins -->
+							<div class="row mx-3 py-4">
+								<div class="anchor" id="all-plugins-section"></div>
+								<h5 class="py-3">All Plugins</h5>
+
+							</div>
+
+						</div>
+
+						<!-- Plugins in categories -->
+            {{#allPossibleCategories}}
+              <div class="tab-pane fade show" id="nav-{{name}}" role="tabpanel"
+              aria-labelledby="nav-{{name}}-tab" tabindex='0'>
+								<div class="row mx-3 py-4">
+								</div>
+							</div>
+            {{/allPossibleCategories}}
+
+					</div>
+				</div>
+			</div>
+		</div>
+	</main>
+</body>
+
+</html>
+
+<style>
+  :root {
+    --bs-body-bg: #ebebeb;
+  }
+  .navbar{
+    padding: 20px 40px 20px 40px;
+  }
+  .search-field {
+    position: relative;
+  }
+  
+  .search-icon {
+    position: absolute;
+    padding: 10px;
+    pointer-events: none;
+    left: 0px;
+  }
+  .search-input {
+    padding-left: 30px;
+  }
+  .banner {
+		padding-top: 4rem;
+		padding-bottom: 4rem;
+	}
```

Comment:
Please pay attention to indentation and mix of tabs and spaces. This is basic thing that should be correct. You can also install a linter to take care of that automatically

Reply:
I've added esllint and prettier to format code, it should all be tabs now.